script/upload.py

```diff
@@ -96,6 +96,8 @@ def yield_brave_packages(dir, channel, version):
         if re.match(r'Brave-Browser-' + channel.capitalize() + r'.*\.dmg$', file):
           yield file
       elif PLATFORM == 'linux':
+        if channel == 'dev':
+           channel = 'unstable'
```

Comment:
Ah, afaik, we uses `dev` instead of `unstable` officially.
Does this change visible to users?

Reply:
Currently the files that are created on linux are named with 'unstable' in the name, here's an example of what I see in last evening's build:

```
ubuntu@ip-172-31-42-80:/var/lib/jenkins/workspace/brave-browser-build-linux/src/out/Release$ ls -ltr brave-browser*
-rw-rw-r-- 1 ubuntu ubuntu 58902808 Jul 17 14:45 brave-browser-unstable-0.50.11-1.x86_64.rpm
-rw-r--r-- 1 ubuntu ubuntu 58845078 Jul 17 14:45 brave-browser-unstable_0.50.11_amd64.deb
-rw-rw-r-- 1 ubuntu ubuntu      106 Jul 17 15:24 brave-browser-unstable_0.50.11_amd64.deb.sha256sum
-rw-rw-r-- 1 ubuntu ubuntu      109 Jul 17 15:24 brave-browser-unstable-0.50.11-1.x86_64.rpm.sha256sum
```

Should we be seeing files with 'dev' in the name?

Reply:
Yep, I think `dev` is our official channel name. @bbondy ?

Reply:
@mbacchi , I think renaming should be done by `yarn create_dist`.
In `lib/createDist.js` there is `renameLinuxDistr` function that not used for now.
If rename is done there, this pr is not needed.

Reply:
How about https://github.com/brave/brave-browser/pull/581 instead of replacing here?

Reply:
sgtm

Reply:
I don't think it should say unstable, I think it should show dev.

Reply:
I've approved brave/brave-browser#581 so I will close this PR